### ISSUE TITLE
unit test framework

### ISSUE THREAD

**Issue by struktured** [KitFreddura/OCaml-Primes#7]
needs one. badly. @KitFreddura - I'm indifferent. Can we settle on one for this project so tests can be written for the critical functions?

**Comment by kitfre** [comment]
I agree, but I'm not too family with any frameworks for ocaml, is there a version of quickcheck? Thanks for all your help, school has been hectic lately so I don't have much time to work on this project and I appreciate all of your work.
